src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Introduce.scss';
+
+const Introduce = () => {
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+  }
+
+  const handleInput = () => {
+    console.log('input 태그 동작');
+  }
+
+  const handleClick = () => {
+    console.log('button 태그 동작');
+  }
+
+  return(
+    <div className="container">
+      <div className="card">
+        <span>
+          <div className='card-image' style={{ backgroundImage: "url('https://image.rocketpunch.com/company/99609/wecode_logo_1590553949.jpg?s=400x400&t=inside')"}} />       
+          <div className="name">김정환</div>
+          <div className="gen-no">44기</div>
+          <form onSubmit={handleSubmit}>   
+            <input type="text" name="input" id="" onInput={handleInput} />
+            <Link to="/product">
+              <button onClick={handleClick}>클릭</button> 
+            </Link>
```

Comment:
```suggestion
              <button onClick={handleClick}>클릭</button> 
```

과제 내용을 보시면 button 태그를 눌렀을 경우,
useNavigate 훅을 활용한 방법으로 페이지 이동을 요구했습니다!

해당 요구사항에 맞게 다시 구현해주세요!